src/Sulu/Bundle/MediaBundle/Entity/Collection.php

```diff
@@ -88,12 +88,12 @@ class Collection implements CollectionInterface, PermissionInheritanceInterface
     private $key;
 
     /**
-     * @var DoctrineCollection<int, CollectionMeta>
+     * @var DoctrineCollection<array-key, CollectionMeta>
      */
     private $meta;
 
     /**
-     * @var DoctrineCollection<int, MediaInterface>
+     * @var DoctrineCollection<array-key, MediaInterface>
```

Comment:
same as above

Reply:
```suggestion
     * @var DoctrineCollection<int, MediaInterface>
```

Still go with int here, phpstan can currently not detect it, but it is always an int.